package.json

```diff
@@ -42,9 +42,12 @@
     "use-breakpoints-width": "^1.0.8"
   },
   "devDependencies": {
+    "@arkweid/lefthook": "^0.7.2",
+    "@trivago/prettier-plugin-sort-imports": "^1.4.1",
     "babel-eslint": "^10.1.0",
     "cypress": "^4.5.0",
     "eslint": "^6.8.0",
+    "eslint-config-halo-lab": "^1.0.12",
     "eslint-config-prettier": "^6.11.0",
```

Comment:
we don't need this anymore and maybe some other eslint packages can be killed

Reply:
fixed